### DISCUSSION TITLE
Command Bookmarking tool

### DISCUSSION THREAD

**Discussion by Jack-Chronicle** [charmbracelet/inspo#36] [Ideas]
Would be amazing to see a way to bookmark your most used commands or just rather long ones

Could be done with gum I think, or in a similar way to Maas' nap, or alfiankans commander but inputting commands into the terminal and optionally running them afterward

**Comment by nkxxll** [comment]
If you want to make it real let's build it I'm down @Jack-Chronicle. I think gum would be a good choice for the use case, what do you think?
